src/pages/messages/components/TicketDateIntervalFilter.tsx

```diff
@@ -23,24 +23,33 @@ export const TicketDateIntervalFilter = () => {
   const selectedDatesParam = searchParams.get(TicketSearchParam.CreatedDate);
   const [selectedFromDate, selectedToDate] = selectedDatesParam ? selectedDatesParam.split(',') : ['', ''];
 
-  const onChangeFromDate = (newDate: Date | null) => {
-    const newFromDate = newDate ? formatDateStringToISO(newDate) : '';
-    if (!newFromDate && !selectedToDate) {
+  const hasValidYear = (date: string) => {
+    return date === '' || date.match(/^\d{4}-/);
```

Comment:
Syns det er rart at `hasValidYear` returnerer `true` når `date ==== ''` 🤔 
```suggestion
    return date.match(/^\d{4}-/);
```